src/builtin.rs

```diff
@@ -0,0 +1,224 @@
+use std::collections::BTreeMap;
+use std::string::String;
+use std::vec::Vec;
+use std::fs::{self, File};
+use std::io::{stdout, Read, Write};
+use std::env;
+use std::process;
+use std::thread;
+
+use super::to_num::ToNum;
+use super::set_var;
+use super::input_editor::readln;
+
+pub fn cat(args: &Vec<String>) {
```

Comment:
Agreed on all. I will leave it in for now, but I am looking forward to removing all the core-utils functions.